ci/terraform/oidc/doc-app-authorize.tf

```diff
@@ -34,7 +34,12 @@ module "doc-app-authorize" {
     DOC_APP_JWKS_URL                   = var.doc_app_jwks_endpoint
     DOC_APP_ENCRYPTION_KEY_ID          = var.doc_app_encryption_key_id
     DOC_APP_TOKEN_SIGNING_KEY_ALIAS    = local.doc_app_auth_key_alias_name
+    DOC_APP_API_ENABLED                = var.doc_app_api_enabled
+    DOC_APP_DOMAIN                     = var.doc_app_domain
+    DOC_APP_DECOUPLE_ENABLED           = var.doc_app_decouple_enabled
     INTERNAl_SECTOR_URI                = var.internal_sector_uri
+    CUSTOM_DOC_APP_CLAIM_ENABLED       = var.custom_doc_app_claim_enabled
+    SUPPORT_AUTH_ORCH_SPLIT            = var.support_auth_orch_split
```

Comment:
We need to add the Doc App variables to the `authorize.tf` instead of this file as it's the `AuthorisationHandler.java` that we are adding the functionality too